package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "strange-forms",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
No need to do this bump– simpler to handle it using `npm version` at release time.  And of course there aren't multiple PRs going into the same release in this case, but it also helps deal with that.